packages/webapp/webapp_server.js

```diff
@@ -168,9 +168,7 @@ WebApp.categorizeRequest = function(req) {
 
   // TODO Perhaps one day we could infer Cordova clients here, so that we
   // wouldn't have to use prefixed "/__cordova/..." URLs.
-  const preferredArchOrder = isModern(browser)
-    ? ['web.browser', 'web.browser.legacy']
-    : ['web.browser.legacy', 'web.browser'];
+  const preferredArchOrder = ['web,browser'];
```

Comment:
Typo: should be `.`, not `,`. Did this not break anything?

Reply:
Fixed